src/KafkaFlow.Retry.SqlServer/Deploy/01 - Create_Tables.sql

```diff
@@ -130,6 +130,41 @@ BEGIN
 END
 GO
 
+IF(NOT EXISTS(SELECT *
+              FROM sys.types
+              WHERE is_table_type=1 AND name='TY_RetryQueueItemsIds'))
+BEGIN
+    CREATE TYPE [dbo].[TY_RetryQueueItemsIds]
+    AS 
+    TABLE
+    (
+     Id BIGINT
```

Comment:
Actually, an improvement to consider for the future: set `Id` as a primary key. It will index the `Id` and potentially makes the queries using this type faster. Obviously, this needs to be tested, don't take this for granted!

Reply:
and in addition to the primary key, we can also set the field as `not null`